src/utils/stages_states.ts

```diff
@@ -125,10 +126,20 @@ const getKaiState = (
     );
     if (_.isNil(s)) return;
     const [_stageName, _stateName, kaiStates] = s;
-    const kaiState = _.find(
-        kaiStates as StateKaiType[],
-        (state) => runUrl === state.broker_msg_body.run.url,
-    );
+    const kaiState = _.find(kaiStates as StateKaiType[], (state) => {
+        /* this is broken and should be removed */
```

Comment:
Is there a ticket or issue we could link to so we know when it's safe to remove this work-around?

Reply:
@jimbair this is not work-around.
This is well known issue.
We talk about it I think each dashboard meeting.
The ticket for this is : https://issues.redhat.com/browse/OSCI-3376